media_entity_audio.module

```diff
@@ -0,0 +1,44 @@
+<?php
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Entity\Entity;
+use Drupal\file\Entity\File;
+use Drupal\Component\Utility\SafeMarkup;
+use Drupal\media_entity_audio\Plugin\Field\FieldFormatter\AudioPlayerHTML5;
+
+/** 
+* Implements hook_theme().
+*/
+function media_entity_audio_theme() {
+  return array(
+    'media_file_formatter' => array(
```

Comment:
Ah... sorry for geeky talk. :)

theme hook should be prefixed with module namespace (media_entity_audio in our case) to prevent collisions with other modules.